src/filter.js

```diff
@@ -73,33 +73,23 @@ define(function (require) {
    *  }
    *  </code></div>
    */
-  p5.Filter = function(type) {
-    this.ac = p5sound.audiocontext;
 
-    this.input = this.ac.createGain();
-    this.output = this.ac.createGain();
 
-    /**
-     *  The p5.Filter is built with a
-     *  <a href="http://www.w3.org/TR/webaudio/#BiquadFilterNode">
-     *  Web Audio BiquadFilter Node</a>.
-     *  
-     *  @property biquadFilter
-     *  @type {Object}  Web Audio Delay Node
-     */
+
+  p5.Filter = function (type) {
+    p5.Effect.call(this, 'Filter');
```

Comment:
I don't think we do anything with this 'Filter' argument, so it can probably be removed?

Also, a convention from other modules is to explicitly require dependencies. So we'd require p5 Effect at the top of this file, like `var Effect = require('effect')` and then refer to it as `Effect` within this module. That way this module isn't making any assumptions about whether p5.Effect already exists. With that approach we can remove `require('effect') from src/app.js as it'll get required by the modules that actually use it. I think the grunt task will result in similar output either way. Do you think that's a good approach?